nodes.py

```
@@ -1240,6 +1506,83 @@ def post(self, shared, prep_res, exec_res):
+    """
+    Compares CSV and API execution results, flags discrepancies, and reconciles values.
+    """
+
+    def prep(self, shared):
+        return {
+            "csv_result": shared.get("csv_exec_result"),
+            "api_result": shared.get("api_exec_result"),
+            "entity_ids": shared.get("entity_ids", {}),
+        }
+
+    def _compare_scalars(self, csv_value, api_value):
```

Ruff (0.14.10)

Missing return type annotation for private function `_compare_scalars`

(ANN202)